src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlBatchCommand.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Data;
+using System.Data.Common;
+using Microsoft.Data.Common;
+
+namespace Microsoft.Data.SqlClient
+{
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+
+    public partial class SqlBatchCommand : DbBatchCommand
+    {
+        private string _text;
+        private CommandType _type;
+        private SqlParameterCollection _parameters;
+        private CommandBehavior _behavior;
+        private int _recordsAffected;
+        private SqlCommandColumnEncryptionSetting _encryptionSetting;
+
+        public SqlBatchCommand()
+        {
+            _type = CommandType.Text;
+        }
+
+        public SqlBatchCommand(string commandText, CommandType commandType = CommandType.Text, IEnumerable<SqlParameter> parameters = null, SqlCommandColumnEncryptionSetting columnEncryptionSetting = SqlCommandColumnEncryptionSetting.UseConnectionSetting)
```

Comment:
Please include samples in 'samples' folder to show all possible usages of the new APIs.

Reply:
I added a sample that shows the general usage pattern for SqlBatch and linked it multiple places in the docs. I didn't add exhaustive samples as they would be duplicative of already well-known properties and methods on SqlCommand.